### PR TITLE
Updated versions in GitHub workflow and updated supported versions

### PR DESCRIPTION
**Pull request by GitRon**
Hi @lociii & @costela 

I just updated the docs for PyPI and the GitHub actions. Let's see if it runs through 🙃

### CODE REVIEW

**Comment by lociii** [comment]
I'd be fine in removing Python 3.7 support completely as it's EOL in 3 months anyways.

File: .github/workflows/workflow.yaml [hunk 1]
```diff
@@ -9,14 +9,14 @@ env:
 
 jobs:
   flake8:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-23.04
```
Comment (costela) [inline .github/workflows/workflow.yaml:12]: I think it's generally safer (in the sense of "principle of least surprise") to use LTS versions?

File: .github/workflows/workflow.yaml [hunk 2]
```diff
@@ -9,14 +9,14 @@ env:
...
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v2
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: 3.1
```
Comment (costela) [inline .github/workflows/workflow.yaml:19]: don't you mean this?
```suggestion
          python-version: "3.10"
```

File: .github/workflows/workflow.yaml [hunk 3]
```diff
@@ -26,7 +26,7 @@ jobs:
 
   test:
     name: Test and coverage
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-23.04
```
Comment (costela) [inline .github/workflows/workflow.yaml:29]: dito: LTS is probably better?

File: .github/workflows/workflow.yaml [hunk 4]
```diff
@@ -36,16 +36,20 @@ jobs:
           - "3.8"
           - "3.9"
           - "3.10"
+          - "3.11"
```
Comment (costela) [inline .github/workflows/workflow.yaml:39]: as @lociii said, we can probably drop 3.7 here :wink:

File: .github/workflows/workflow.yaml [hunk 5]
```diff
@@ -36,16 +36,20 @@ jobs:
...
+          - "4.1"
         database:
           - sqlite
           - mysql
           - postgres
         exclude:
           - python-version: 3.7
             django: 4.0
+          - python-version: 3.7
+            django: 4.1
```
Comment (costela) [inline .github/workflows/workflow.yaml:52]: if we do drop 3.7, we can drop these too, right?

File: .github/workflows/workflow.yaml [hunk 6]
```diff
@@ -135,15 +142,15 @@ jobs:
     if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
     name: Build and publish to PyPI
     needs: test
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-23.04
```
Comment (costela) [inline .github/workflows/workflow.yaml:145]: LTS :wink:

File: setup.py [hunk 7]
```diff
@@ -73,15 +73,16 @@ def get_version(*file_paths):
         'Framework :: Django',
         'Framework :: Django :: 3.2',
         'Framework :: Django :: 4.0',
+        'Framework :: Django :: 4.1',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: BSD License',
         'Natural Language :: English',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
```
Comment (costela) [inline setup.py:81]:
```suggestion
```

**Review by costela — CHANGES_REQUESTED** [review]
Thanks as always for the work! :medal_sports: 

Just some minor notes :wink:

**Comment by GitRon** [comment]
@lociii @costela Good remarks, updated the PR!

**Comment by GitRon** [comment]
Any idea why the pipeline is not running through? 🤔

**Comment by GitRon** [comment]
Found something, we'll see if it helps.

**Comment by lociii** [comment]
LGTM now. Great work as always. Thanks for keeping the setup up to date.

**Comment by GitRon** [comment]
Thx! No problem. I hope that we get this package to the next level - together 🙃

**Pull request merged**